permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsController.kt

```diff
@@ -80,6 +81,13 @@ actual class PermissionsController(
                 Manifest.permission.BLUETOOTH,
                 Manifest.permission.ACCESS_COARSE_LOCATION
             )
+            Permission.REMOTE_NOTIFICATION -> {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                    listOf(Manifest.permission.ACCESS_NOTIFICATION_POLICY)
```

Comment:
Android not require runtime permissions for notifications. Just return empty list. Current version request incorrect permission - it access to change notifications of other apps

Reply:
Ok, will change with Permission.REMOTE_NOTIFICATION -> emptyList()